src/game_logic/unit.ts

```diff
@@ -47,7 +47,7 @@ class Unit {
         this.hasChronoUntilFrame = -1
         // Zerg townhalls
         this.hasInjectUntilFrame = -1
-        this.nextLarvaSpawn = -1
+        this.nextLarvaSpawn = 10.71428571 * 22.4
```

Comment:
Could you please add a comment on where you have that number from? 
I have that number from liquipedia I believe https://liquipedia.net/starcraft2/Larva_(Legacy_of_the_Void)
> at a rate of 1 Larva every 11 seconds.

Reply:
Yeah, it's hard finding a reference for what that number is supposed to be.  The liquipedia link was the only place I saw 11.  I saw a couple places that mentioned 15 seconds at normal speed which matched with what I saw in game.  Since faster is 1.4 times faster, the 15/1.4=10.7 and some change.  This also got the app to line up with the game where it was slowing drifting before.
https://gaming.stackexchange.com/questions/59831/how-quickly-do-larvae-spawn-in-starcraft-2
Personally, I trust the experimental evidence I got from in game over what some random person put online, though.  If you doubt me on the number, you could always do the same thing.
As for the comment, would you prefer a comment linking to the really old stackexchange question or just put the 15/1.4 math in a comment?

Reply:
15 / 1.4 math as a comment is good enough, thank you!